tests/data/dataset_readers/SQUAD_SAMPLE_FILE.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
yeah, the thing that i was struggling with is that json doesn't allow line breaks within a field, unless you use `\n`, which means that I'd have to have the passage all on one line. i think it works to define them above and reference them as variables, though...